opentelemetry-exporters-newrelic-auto/src/main/java/com/newrelic/telemetry/opentelemetry/export/auto/NewRelicSpanExporterFactory.java

```diff
@@ -0,0 +1,33 @@
+package com.newrelic.telemetry.opentelemetry.export.auto;
+
+import com.newrelic.telemetry.Attributes;
+import com.newrelic.telemetry.opentelemetry.export.NewRelicSpanExporter;
+import io.opentelemetry.sdk.contrib.auto.config.Config;
+import io.opentelemetry.sdk.contrib.auto.config.SpanExporterFactory;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+public class NewRelicSpanExporterFactory implements SpanExporterFactory {
+
+  private static final String NEWRELIC_API_KEY = "newrelic.api.key";
+  private static final String NEWRELIC_SERVICE_NAME = "newrelic.service.name";
+  private static final String DEFAULT_NEWRELIC_SERVICE_NAME = "(unknown service)";
+
+  /**
+   * Creates an instance of a {@link SpanExporter} based on the provided configuration.
+   *
+   * @param config The configuration
+   * @return An implementation of a {@link SpanExporter}
+   */
+  @Override
+  public SpanExporter fromConfig(Config config) {
+    final String apiKey = config.getString(NEWRELIC_API_KEY, "");
+    // todo: update key to service.name once it is standardized
```

Comment:
I think the TODO will actually be to remove service.name altogether from the exporter, but this is fine to at least remind us to keep track of how things evolve.

Reply:
Gotcha. Updated the verbiage now